### PR TITLE
Update footer credit

### LINKED ISSUE #693: Footer credit text improvement
The default text at the bottom of every page is "Storefront designed by WooCommerce." While this makes sense for Woo experts, for everyone else this sounds like WooCommerce designed and/or runs the site, which is misleading. 

I think this should be changed to something more specific, such as "Powered with Storefront by WooCommerce" or "Built with Storefront by WooCommerce."

### PR DESCRIPTION
**Pull request by tiagonoronha**
Updates footer credit from "Storefront designed by WooCommerce" to "Built with Storefront by WooCommerce".

Closes #693.

### CODE REVIEW

**Comment by GlennMartin1** [comment]
Save a redirect: The link should point to https://woocommerce.com

**Comment by MarinaWoo** [comment]
Built with Storefront & WooCommerce +1

File: inc/storefront-template-functions.php [hunk 1]
```diff
@@ -127,7 +127,20 @@ function storefront_credit() {
 		<div class="site-info">
 			<?php echo esc_html( apply_filters( 'storefront_copyright_text', $content = '&copy; ' . get_bloginfo( 'name' ) . ' ' . date( 'Y' ) ) ); ?>
 			<?php if ( apply_filters( 'storefront_credit_link', true ) ) { ?>
-			<br /> <?php printf( esc_attr__( '%1$s designed by %2$s.', 'storefront' ), 'Storefront', '<a href="http://www.woocommerce.com" title="WooCommerce - The Best eCommerce Platform for WordPress" rel="author">WooCommerce</a>' ); ?>
+			<br /> <?php printf(
+					wp_kses(
+						__( 'Built with <a href="%1$s" target="_blank" title="WooCommerce - The Best eCommerce Platform for WordPress" rel="author">Storefront & WooCommerce</a>.', 'storefront' ),
```
Comment (mikejolley) [inline inc/storefront-template-functions.php]: Should use html `&amp;` in the string.

Comment (mikejolley) [inline inc/storefront-template-functions.php]: I'd remove the HTML entirely from this string - use placeholders for the opening and closing tag. This means translators won't bugger it up, and it means we don't need to translate our SEO title tag thing in there. It also means we can use esc_html and not rely on kses :)

**Pull request merged**